src/fee-accounts/containers/overview/account-overview.component.ts

```diff
@@ -12,33 +13,22 @@ export class AccountsOverviewComponent implements OnInit, OnDestroy {
   columnConfig: GovukTableColumnConfig[];
   tableRows: {}[];
   feeAccountsSubscription: Subscription;
+  accounts$: Observable<Array<any>>;
```

Comment:
maybe would be good to create model for this

Reply:
Good spot.